nebula1/visor_run_config.toml

```diff
@@ -0,0 +1,23 @@
+name = "<insert_vega_version_here>"
```

Comment:
Can we set an actual value here?

Reply:
yeah. I have updated to genesis. I think it is just name which is not used anywhere.